view/next-project/src/stories/teacher/AddModal.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import { Meta, StoryFn } from '@storybook/react';
+import { SetStateAction } from 'react';
+import { AddModal } from '@components/teacher';
+import { Department } from '@type/common';
+
+export default {
+  title: 'FinanSu/teacher/AddModal',
+  component: AddModal,
+  argTypes: {},
```

Comment:
```suggestion
  argTypes: {
    setShowModal: {
      control: 'boolean',
      defaultValue: false,
    },
  },
```
booleanを切り替えできるようにしたいです。